Configure

```diff
@@ -490,6 +490,7 @@ my @disablables = (
     "ssl",
     "ssl-trace",
     "static-engine",
+    "stats",
```

Comment:
Do we want this to be "stats" or "lhash-stats" or similar?

Currently, only lhash is impacted and I don't expect that to change.

ALso note that we collect TLS stats and will be collecting QUIC stats (which cannot be turned off).  So calling this just "stats" is a bit of a misnomer IMO.